app/views/conference/_conference_details.html.haml

```diff
@@ -26,7 +26,7 @@
             - if conference.registration_open?
               - if conference.user_registered?(current_user)
                 = link_to "My Registration", conference_conference_registrations_path(conference.short_title), :class =>"btn btn-default"
-              - else
+              - elsif not conference.registration_limit_exceeded?
```

Comment:
Please use `!conference.registration_limit_exceeded?` instead of `not`
http://ruby-doc.org/core-2.1.3/doc/syntax/precedence_rdoc.html